src/rpc/util.cpp

```diff
@@ -4,11 +4,9 @@
 
 #include <base58.h>
 #include <keystore.h>
-#include <pubkey.h>
 #include <rpc/protocol.h>
 #include <rpc/util.h>
 #include <tinyformat.h>
-#include <utilstrencodings.h>
```

Comment:
`utilstrencodings.h` seems to be included in `rpc/util.h` for no reason. Maybe it's better to remove it there?